packages/conf-binutils/conf-binutils.0.2/files/find-binutils.ml.in

```diff
@@ -122,18 +130,51 @@ let collect_targets objdumps =
 
 let which = cmd "which %s"
 
+let of_env var =
+  try Some (Sys.getenv var)
+  with Not_found -> None
+
+let find_in_cellar name =
+  match cmd "brew --cellar" with
+  | None -> None
+  | Some cellar ->
+    take_first "find %s/binutils -name %s" cellar name
+
+let of_opam_config = function
+  | "" -> None
+  | x  -> Some x
+
+let rec find_map xs =
```

Comment:
it is not `find_map` it is just `first_some`, and it is much better to express it as `List.find is_some`